src/bioregistry/data/bioregistry.json

```diff
@@ -4133,21 +4133,27 @@
   "biorxiv": {
     "description": "The bioRxiv is a preprint server for biology",
     "example": "2021.07.23.453588",
+    "example_extras": [
+      "000091",
+      "000091v1",
+      "2022.01.24.477526v1"
+    ],
     "go": {
       "homepage": "https://www.biorxiv.org/",
       "name": "bioRxiv",
       "prefix": "bioRxiv",
       "synonyms": [
         "BIORXIV"
       ],
-      "uri_format": "https://www.biorxiv.org/content/$1"
+      "uri_format": "https://www.biorxiv.org/content/10.1101/$1"
```

Comment:
Ahhhh I totally missed this, you edited the imported URI format from GO, not the top-level record. Will fix no!

Reply:
Fixed in https://github.com/biopragmatics/bioregistry/commit/9d8bf7cb97f43e5af5204a89358fde5385fc185f

Reply:
Ah shoot! My bad. Didn't realize this file had the imported registry records.